services/transact-faas/package.json

```diff
@@ -12,6 +12,8 @@
     "deploy:dev": "yarn run zip && yarn run update:faas:dev"
   },
   "dependencies": {
+    "aws-sdk": "^2.395.0",
```

Comment:
Unless version > 2.290.0 required, no need to add `aws-sdk` as local dependency: https://docs.aws.amazon.com/lambda/latest/dg/current-supported-versions.html